File: iraf/unix/boot/spp/rpp/rppfor/push.f
      integer function push (ep, argstk, ap)
      integer ap, argstk (100), ep
      if (.not.(ap .gt. 100))goto 23000
      call baderr (19Harg stack overflow.)
23000 continue
      argstk (ap) = ep
      push = ap + 1
      return
      end

File: unix/boot/spp/rpp/rpprat/push.r
#-h-  push			  249  local   12/01/80  15:54:34
# push - push ep onto argstk, return new pointer ap
   include  defs

   integer function push (ep, argstk, ap)
   integer ap, argstk (ARGSIZE), ep

   if (ap > ARGSIZE)
      call baderr ('arg stack overflow.')
   argstk (ap) = ep
   push = ap + 1
   return
   end
